pkg/apis/logging/v1/output_types.go

```diff
@@ -136,3 +138,23 @@ type Elasticsearch struct {
 	// +optional
 	StructuredTypeName string `json:"structuredTypeName,omitempty"`
 }
+
+// Loki provides optional extra properties for `type: loki`
+type Loki struct {
```

Comment:
General question, are we omitting the ability for user to set the Tenant ID for the simple forwarding case in custom managed Loki installation. If yes, why?

Reply:
I think I misunderstood a conversation we had. We talked about it being sufficient to use the URL path for our own managed Loki. I assumed that extended to all Loki installations. Should I re-instate that ability for user-managed Lokis?

Reply:
Yes user-managed Loki's should support the TenantID field.

Reply:
/hold
Restoring tenantID field.